base/setup/reactos/lang/en-US.rc

```diff
@@ -173,3 +173,30 @@ BEGIN
     IDS_PARTITION_SIZE "Size"
     IDS_PARTITION_STATUS "Status"
 END
+
+/* Progress Strings */
+
+STRINGTABLE
+BEGIN
+    ///// IDS_PREPARE_PARTITIONS "Preparing partitions..."
+    ///// IDS_PREPARE_FILES "Preparing the list of files to be copied, please wait..."
+    ///// IDS_COPYING_FILES "Copying the files..."
```

Comment:
```suggestion
    //IDS_PREPARE_PARTITIONS "Preparing partitions..."
    //IDS_PREPARE_FILES "Preparing the list of files to be copied, please wait..."
    //IDS_COPYING_FILES "Copying the files..."
```

Reply:
Yep, these were markers for me to indicate where the stuff from PR #7159 will land to. Will disappear in the final PR version.